rewriter/Prop.cc

```diff
@@ -232,6 +232,8 @@ optional<NodesAndPropInfo> processProp(core::MutableContext ctx, ast::Send *send
                 }
             }
         }
+
+        ret.nodes.emplace_back(ast::MK::Unsafe(send->loc, rules->deepCopy()));
```

Comment:
Note: We tried using keepForIde, but the verifier complained that the keepForIde node lacked loc information.

We then added in an exclusion for keepForIde in that pass, but then it complained that `Constant(loc, core::Symbols::Sorbet_Private_Static())`, which keepForIde creates, lacks loc information.

It seemed like we were fighting an uphill battle, and that T.unsafe might be the safest option.